.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+node_js:
+  - 'stable'
+cache: yarn
+# this is necessary to ensure we build every branch in repo, PR builds will still be triggered
+branches:
+  only:
+  - master
+# we skip auto install step to use parallelized build stage and install in packages
+install: skip
+# stages are sequencial and build will not continue if one fails
+jobs:
+  include:
+    - stage: install 
+      script: yarn
+      script: cd packages/veto-backend && yarn
```

Comment:
Since line 15 already install everything, we don't need to install `veto-backend` & `veto-frontend` separately.

Reply:
also, looks like line 16 is never executed https://travis-ci.com/Veriphi/veto/jobs/285229570 i'm not sure why that is  :(